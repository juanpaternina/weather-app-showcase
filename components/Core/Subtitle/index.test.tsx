import { render, screen } from '@testing-library/react-native';

const { Subtitle } = require('.');

describe('Subtitle', () => {
  test('It should render a Text component', () => {
    const { toJSON } = render(<Subtitle>Subtitle</Subtitle>);
    expect(toJSON()).toMatchSnapshot();
    const text = screen.getByText('Subtitle');
    expect(text).toBeDefined();
  });
});
