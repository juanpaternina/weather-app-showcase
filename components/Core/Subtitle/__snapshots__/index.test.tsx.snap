// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subtitle It should render a Text component 1`] = `
<Text
  style={
    {
      "color": "black",
      "fontFamily": "medium",
      "fontSize": 20,
      "fontWeight": "bold",
    }
  }
>
  Subtitle
</Text>
`;
